app/queries/servers/channel.nomock.test.ts

```diff
@@ -0,0 +1,374 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
The reason for this file to exist is that the `channel.test.ts` was giving me problems with the mocking, while these tests don't use any mock.

I moved here the observeMyChannelUnreads while at it, since they also use no mocks.